README.md

```diff
@@ -8,6 +8,121 @@
   yarn add @artsy/react-responsive-media
 ```
 
+## Basic Usage
+
+```js
+import { createMedia } from "@artsy/fresnel"
+
+const ExampleAppMedia = createMedia({
+  breakpoints: {
+    xs: 0,
+    sm: 768,
+    md: 1000,
+    lg: 1200,
+  },
+})
+
+const { Media, MediaContextProvider } = ExampleAppMedia
+
+const App = () => {
+  return (
+    <MediaContextProvider>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
+    </MediaContextProvider>
+  )
+}
+```
+
+## Server-side Rendering (SSR) Usage
+
+First, configure `@artsy/fresnel` in a `Media` file that can be shared across the app:
+
+```js
+// Media.tsx
+
+import { createMedia } from "@artsy/fresnel"
+
+const ExampleAppMedia = createMedia({
+  breakpoints: {
+    xs: 0,
+    sm: 768,
+    md: 1000,
+    lg: 1200,
+  },
+  interactions: {},
+})
+
+export const { Media, MediaContextProvider, createMediaStyle } = ExampleAppMedia
+```
+
+Create a new `App` file which will be the launching point for our application:
+
+```js
+import React from "react"
+import { Media, MediaContextProvider } from "./Media"
+
+export const App = () => {
+  return (
+    <MediaContextProvider>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
+    </MediaContextProvider>
+  )
+}
+```
+
+Mount `<App />` on the client:
+
+```js
+// client.tsx
+
+import React from "react"
+import ReactDOM from "react-dom"
+import { App } from "./App"
+
+ReactDOM.render(<App />, document.getElementById("react"))
+```
+
+Then on the server, setup SSR rendering and call `createMediaStyle`, injecting the necessary css into the header:
+
+```js
+// server.tsx
+
+import React from "react"
+import ReactDOMServer from "react-dom/server"
+import express from "express"
+
+import { App } from "./App"
+import { createMediaStyle } from "./Media"
+
+const app = express()
+
+app.get("/", (_req, res) => {
+  const html = ReactDOMServer.renderToString(<App />)
+
+  res.send(`
+    <html>
+      <head>
+        <title>@artsy/fresnel - SSR Example</title>
+        <style type="text/css">${createMediaStyle()}</style>
```

Comment:
Let’s move this to a constant at the module scope level, as it doesn’t change and thus doesn’t need to be performed more than once and probably better illustrates the key take-away.